integration_tests/src/main/python/data_gen.py

```diff
@@ -244,7 +244,7 @@ def start(self, rand):
 
 class DecimalGen(DataGen):
     """Generate Decimals, with some built in corner cases."""
-    def __init__(self, precision=None, scale=None, nullable=True, special_cases=None, avoid_positive_values=False):
+    def __init__(self, precision=None, scale=None, nullable=True, special_cases=None, avoid_positive_values=False, full_precision=False):
```

Comment:
Could we add a comment/doc string about what `full_precision` means? It is not clear from just the name of it.

Reply:
done.